src/components/ChartPriceContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+
+import ChartPrice from './ChartPrice'
+
+export default class ChartPriceContainer extends Component {
+
+  static defaultProps =  {
```

Comment:
this defaultProps, must be in `ChartPrice`. Sorry for not having expressed it correctly when I explained